pkg/training/trainer_tensorflow.go

```diff
@@ -433,17 +422,31 @@ func makeJobStatusSortedByTime(conditions []commonv1.JobCondition) []commonv1.Jo
 	return []commonv1.JobCondition(newConditions)
 }
 
-func checkStatus(status commonv1.JobStatus) commonv1.JobConditionType {
-	t := commonv1.JobConditionType("Pending")
+// getStatus returns the status of the Job i.e. PENDING, QUEUING, RUNNING, SUCCEEDED and FAILED.
+func getStatus(status commonv1.JobStatus) string {
+	var s string
+	if hasCondition(status, commonv1.JobSucceeded) {
+		s = string(commonv1.JobSucceeded)
+	} else if hasCondition(status, commonv1.JobFailed) {
+		s = string(commonv1.JobFailed)
+	} else if hasCondition(status, commonv1.JobConditionType("Queuing")) {
```

Comment:
I think If a job meets multiple conditions (e.g., both Queuing and Running), the order in getStatus() may return an incorrect status.

Reply:
I think if it's possible in logic, we should handle this.